REVERT: https://github.com/WebKit/WebKit/commit/28d9d62aa2662266c2d9cb0032b036a257c6c8ab
This prevents a crash at exit time for any application using WebkitGTK.
From 28d9d62aa2662266c2d9cb0032b036a257c6c8ab Mon Sep 17 00:00:00 2001
From: Pablo Saavedra <psaavedra@igalia.com>
Date: Wed, 23 Feb 2022 09:32:13 +0000
Subject: [PATCH] [GTK][WPE] PlatformDisplay::terminateEglDisplays() is never called https://bugs.webkit.org/show_bug.cgi?id=217655

Index: Source/WebKit/WebProcess/WebProcess.h
--- Source/WebKit/WebProcess/WebProcess.h.orig
+++ Source/WebKit/WebProcess/WebProcess.h
@@ -511,7 +511,7 @@ class WebProcess : public AuxiliaryProcess (private)
     bool shouldTerminate() override;
     void terminate() override;
 
-#if USE(APPKIT) || PLATFORM(GTK) || PLATFORM(WPE)
+#if USE(APPKIT)
     void stopRunLoop() override;
 #endif
 
